### DISCUSSION TITLE
Meeting 2024-09-10

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#793] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-09-10)

### Announcements
* None yet

**Comment by jannic** [comment]
Today's list of pull requests that could be reviewed / approved / merged:
- https://github.com/rust-embedded/cortex-m/pull/557
- https://github.com/rust-embedded/cortex-m/pull/535
- https://github.com/rust-embedded/cortex-m/pull/500

**Reply by adamgreig** [reply]
Thanks for triaging!

**Comment by adamgreig** [comment]
I'll be away and unable to run the meetings on the 17th and 24th Sept and 1st Oct

**Reply by adamgreig** [reply]
Looks like @jamesmunns may be able to help on the 24 and 1st, and possibly @BartMassey on the 17th.

**Comment by adamgreig** [comment]
[Implement core::error::Error in embedded-hal](https://github.com/rust-embedded/embedded-hal/pull/630)

**Comment by adamgreig** [comment]
[critical-section v1.1.3 breaks semver rules?](https://github.com/rust-embedded/critical-section/issues/51)

**Reply by adamgreig** [reply]
We concluded that yes, it should have been 1.2.0, but doing so wouldn't have prevented the problem in the issue, and doing so retrospectively also won't fix anything, so we won't make a new release but will try and ensure future releases get the right version bump.
